website/photos/admin.py

```diff
@@ -101,11 +101,10 @@ class PhotoAdmin(admin.ModelAdmin):
     list_display = (
         "__str__",
         "album",
-        "hidden",
         "num_likes",
     )
     search_fields = ("file",)
-    list_filter = ("album", "hidden")
```

Comment:
I think removing the hidden picture feature would be better in a different pull request.